config/default.yml

```diff
@@ -1,6 +1,120 @@
 # Write it!
 
+AllCops:
+  EnabledByDefault: true
+  Exclude:
+    - 'bin/**/*'
+    - 'db/schema.rb'
+    - 'vendor/**/*'
+
+Bundler/OrderedGems:
+  TreatCommentsAsGroupSeparators: false
+
 Fairmoney/NoIfInTestDescriptions:
   Description: 'Test descriptions should not include "if", use "when" instead'
   Enabled: true
   VersionAdded: '0.91'
+
+Layout/CaseIndentation:
+  EnforcedStyle: end
+
+Layout/EndAlignment:
+  EnforcedStyleAlignWith: variable
+
+Layout/LineLength:
+  Max: 149
+
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+  IndentationWidth: 2
+
+Layout/MultilineOperationIndentation:
+  EnforcedStyle: indented
+
+Metrics/BlockLength:
+  Exclude:
+    - 'config/environments/*.rb'
+    - 'config/routes.rb'
+    - 'db/migrate/*'
+    - 'spec/**/*'
+
+Lint/ConstantResolution:
```

Comment:
This is enabled on the bank side I think it can be good to not have it as a default for new projects?